staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -151,7 +151,12 @@ func ValueOrDefault(intOrPercent *IntOrString, defaultValue IntOrString) *IntOrS
 	return intOrPercent
 }
 
-func GetValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool) (int, error) {
+//GetScaledValueFromIntOrPercent is meant to replace GetValueFromIntOrPercent.
+//This method returns a scaled value from an IntOrString type. If the IntOrString
+//is a percentage string value it's treated as a percentage and scaled appropriately
+//in accordance to the total, if it's an int value it's treated as a a simple value and
+//if it is a numeric string value it returns an error.
+func GetScaledValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool) (int, error) {
```

Comment:
You seem to have changed the name of the old function? You need to keep the old function the same name to not accidentally fix load bearing bugs.

Reply:
that is git diffing unfortunately... `GetValueFromIntOrPercent` in this PR maintains the old code by inlining the old getIntOrPercentValue implementation as-is

Reply:
(per my request at https://github.com/kubernetes/kubernetes/pull/89482#discussion_r488027200)

Reply:
oh, I see, the code looks the same because the function it called changed.

Reply:
right, I wanted to contain the badness to the single deprecated exported method